bin/terraform.sh

```diff
@@ -84,7 +84,11 @@ function plan() {
         > "$output"
     retcode="$?"
     set -e
-    $DOCKER_LANDSCAPE - < "$output"
+    if [[ "$retcode" = 0 ]]; then
```

Comment:
It's better to use `-eq` instead of `=` when comparing integers.
See: http://tldp.org/LDP/abs/html/comparison-ops.html
```suggestion
    if [[ "$retcode" -eq 0 ]]; then
```

Reply:
Also, when defining `retcode`, we could specify that it's an integer since we'll only expect integers from `$?`.
So, instead of `local retcode`, we could say `local -i retcode`.